examples/chip-tool/commands/payload/SetupPayloadParseCommand.cpp

```diff
@@ -108,6 +108,14 @@ CHIP_ERROR SetupPayloadParseCommand::Print(chip::SetupPayload payload)
                     }
                     humanFlags.Add("On IP network");
                 }
+                if (payload.rendezvousInformation.Value().Has(RendezvousInformationFlag::kWiFiPAF))
+                {
+                    if (!humanFlags.Empty())
+                    {
+                        humanFlags.Add(", ");
+                    }
+                    humanFlags.Add("WIFIPAF");
```

Comment:
Shouldn't this be more like:
```suggestion
                    humanFlags.Add("Wi-Fi PAF");
```